magento/magento2
# Correcting position of Product name in Thumbnail popup

### Description (*)
When admin user click on Product Thumbnail one popup is showing and there product title is not in the center.


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. login to admin panel -> Catalog -> Products
2. Click on the Thumbnail of any product and check the position of Product's Name

Actual Result (*)
![image](https://user-images.githubusercontent.com/52840038/130439502-ea1d7e01-2066-4f31-bb57-15731046f0c5.png)

Expactced Result (*)
![image](https://user-images.githubusercontent.com/52840038/130439621-647442a3-5a99-4cfa-8da6-760fde26964e.png)

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#34155: Correcting position of Product name in Thumbnail popup
